src/main/java/org/commcare/formplayer/services/InstallService.java

```diff
@@ -81,6 +91,7 @@
             engine.initEnvironment();
             installTimer.end();
             installTimer.record();
+            redisSessionCache.getOperations().delete(storageFactory.getUsername());
```

Comment:
I'm confused, doesn't this cache key require a domain and a potential "as" name, or is his equivalent somehow to that call?